vsphere-cpi.html.md.erb

```diff
@@ -58,7 +58,7 @@ networks:
 vSphere CPI does not support dynamic or vip networks.
 
 ---
-## <a id='resource-pools'></a> Resource Pools / VM Types
+## <a id='resource-pools'></a>VM Types (Resource Pools)
```

Comment:
Since this is a header used across all CPIs, it would be nice if this were updated elsewhere. Or saved for a followup commit?